config/recipes/desktop/recipe-kinoite-main.yml

```diff
@@ -10,10 +10,10 @@ image-version: 39
 # list of modules, executed in order
 # you can include multiple instances of the same module
 modules:
-  - from-file: common-files.yml
-  - from-file: kinoite-packages.yml
-  - from-file: desktop-packages.yml
-  - from-file: common-packages.yml
-  - from-file: common-bling.yml
-  - from-file: common-scripts.yml
+  - from-file: recipes/common/common-files.yml
+  - from-file: recipes/common/kinoite-packages.yml
+  - from-file: recipes/common/non-server-packages.yml
+  - from-file: recipes/common/common-packages.yml
+  - from-file: recipes/common/common-bling.yml
+  - from-file: recipes/common/common-scripts.yml
```

Comment:
this is just my opinion.

placing modules in the `recipes/common/` path makes it seem like they are recipes. Imo the term recipes should be reserved for actual recipe files. So maybe something like `recipes/modules/common` would be a bit tidier but i'm not sure this is just how i feel.